brendenwest/cascade
# Contact Us screen

## Takes care of #19. 
Contents are scrapped from https://cascade.org/about/contact-us.
## New Dependencies and packages added
-  "react-native-maps": (React Native Map components for iOS + Android)
- com.airbnb.android.react.maps.MapsPackage : (An open source package form AirBnb that allows to have a map component on mobile apps)

## To Test

1. git checkout contactUs-screen
2. git pull
3. npm install
4. npx react-native start
5. [New terminal] npx react-native run-android
6. Make sure the app runs without errors!
7. Ensure ALL aspects of the app works correctly

##Screenshots
![image](https://user-images.githubusercontent.com/26123463/142745106-dc441f25-58ca-4531-889f-94b4ec927457.png)
![image](https://user-images.githubusercontent.com/26123463/142745116-4a328209-4d91-414a-a028-965e667c3459.png)
![image](https://user-images.githubusercontent.com/26123463/142745134-23f227e6-c894-43f5-af18-f8dfb06376a2.png)
